app/assets/v2/js/grants/_new.js

```diff
@@ -277,7 +277,26 @@ Vue.mixin({
           _alert(err.message, 'danger');
         }
       });
-    }
+    },
+    onBlur(event) {
+      let vm = this;
+      
+      event.preventDefault();
```

Comment:
You won't need this is you add the prevent to the @blur in the html. https://vuejs.org/v2/guide/events.html#Event-Modifiers